src/main/java/servlets/BillingConfig.java

```diff
@@ -14,36 +14,64 @@
 
 package servlets;
 
+import data.JsonConverter;
+import data.Vendor;
+
+import java.io.File;
 import java.io.IOException;
+import java.util.Enumeration;
+import java.util.Map;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 @WebServlet("/BillingConfig")
 public class BillingConfig extends HttpServlet {
+  private static final String CONTENT_TYPE_TEXT_HTML = "text/html;";
+  private static final String REDIRECT_READFILE = "/index.html";
+
+  public static final String VENDOR_ID = "vendor-id";
+  public static final String ACCOUNT_ID = "account-id";
+  public static final String LEGACY_CUSTOMER_ID = "legacy-customer-id";
+  public static final String NEXT_GEN_CUSTOMER_ID = "next-gen-customer-id";
+  public static final String LEGACY_ACCOUNT_ID = "legacy-account-id";
+  public static final String NEXT_GEN_ACCOUNT_ID = "next-gen-account-id";
+  public static final String CURRENCY_CODE = "currency-code";
+  public static final String DIRECTION = "direction";
+  public static final String ENTITY = "entity";
+  public static final String MATCHING_MODE = "matching-mode";
+  public static final String AGGREGATION_MODE = "aggregation-mode";
```

Comment:
Are these all needed? Does the JsonConverter class handle the field names?

Reply:
BillingConfig, Account, and Vendor classes all use these. I think it makes the most sense here but I don't have any preference